src/registerProviders.ts

```diff
@@ -7,16 +7,37 @@ import { BingImageSearch } from "./providers/storage/bingImageSearch";
 import { LocalFileSystemProxy } from "./providers/storage/localFileSystemProxy";
 import { StorageProviderFactory } from "./providers/storage/storageProvider";
 import registerToolbar from "./registerToolbar";
+import { strings } from "./common/strings";
 
 export default function registerProviders() {
     // Storage Providers
-    StorageProviderFactory.register("localFileSystemProxy", (options) => new LocalFileSystemProxy(options));
-    StorageProviderFactory.register("azureBlobStorage", (options) => new AzureBlobStorage(options));
+    StorageProviderFactory.register({
+        name: "localFileSystemProxy",
+        displayName: strings.connections.providers.local.title,
+        factory: (options) => new LocalFileSystemProxy(options),
+    });
+    StorageProviderFactory.register({
+        name: "azureBlobStorage",
```

Comment:
Is there a central place where we can keep these strings? It would be nice to not need to type out string literals if not necessary. Open to discussion though

Reply:
Let's discuss... Now since the provider list is dynamic there really shouldn't be any other instances of this string outside of unit tests.  Happy to put these into a constant/enum if we think we need it.